bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+
+	conn, err := tls.Dial("tcp", address, conf)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Close()
+
+	_, err = conn.Write(msg)
```

Comment:
I didn't read the underlying `Write` function, but shouldn't we check that we actually wrote the whole message (and not only a part of it)?